src/graph/Graph.tsx

```diff
@@ -17,27 +18,21 @@ import {
   findDownstreamEdges,
 } from './graph-ops'
 
-const Graph2 = ({ data, config /* , chromaticScale */ }: Props) => {
+const GraphComponent = ({ data, config, layoutDirection = 'LR' /* , chromaticScale */ }: Props) => {
```

Comment:
I suggest putting 'layoutDirection' to 'config' to having the same model with other components

Reply:
Yes, we should put it to the config and a chromatic scale as well

Reply:
fixed